src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Ball();
+        Lottery ball2 = new Ball();
+        Lottery ball3 = new Ball();
+
+        Lottery[] ball = {ball1, ball2, ball3};
+        for (Lottery balls : ball) {
+            System.out.println(balls.getRandomBall());
```

Comment:
why are you creating 3 instances of Lottery? Let's use one instance 
```suggestion
        Lottery ball1 = new Ball();
        Lottery ball2 = new Ball();
        Lottery ball3 = new Ball();

        Lottery[] ball = {ball1, ball2, ball3};
        for (Lottery balls : ball) {
            System.out.println(balls.getRandomBall());
```

Reply:
ok, gonna change it now.